website/docs/tutorial/1b-setting-up-snowflake.md

```diff
@@ -206,107 +229,68 @@ There are two ways to connect dbt Cloud and Snowflake. The first option is Partn
+  - **Username:** The username you created for Snowflake. Note: The username is not your email address and is usually your first and last name together in one word. 
+  - **Password:** The password you set when creating your Snowflake account
+  - **Schema:** You’ll notice that the schema name has been auto created for you and is `dbt_` followed by your first initial and last name. This is the schema connected directly to your development environment and is where your models will be built when running dbt within the Cloud IDE.
+  - **Target name:** leave as default
+  - **Threads:** Leave as 4. This is the number of simultaneous connects that dbt Cloud will make to build models concurently.
 
 <p align="center">
 <Lightbox src="/img/snowflake_tutorial/dbt_cloud_snowflake_development_credentials.png" title="dbt Cloud - Snowflake Development Credentials" />
 </p>
 
-9. Scroll back to the top of the page and click "Test" to check your connection with Snowflake. If it’s successful, you should see a green connection succeeded banner at the top of the page and you can continue. If the connection is unsuccessful you should see an error message returned, allowing you to take action to resolve the issue specified by the error. Keep testing until you’re able to successfully create the connection. 
+4. Click “Test” at the top. This will check that dbt Cloud can access your Snowflake account.
+5. If test successful, click “Continue”
 
-10. Now it’s time to create your Git repository. For this setup we’re going to stick with a dbt Cloud managed repository, but feel free to create your own repo and connect it to your git provider of choice if you’d like to manage it yourself. To create the managed repository, make sure that `Managed` is selected from the list of repos/options and then type in the name that you’d like to give the repo. Once complete, click "Create".
+#### Initialize your repository and start development
 
-<p align="center">
-<Lightbox src="/img/snowflake_tutorial/dbt_cloud_set_up_a_repository.png" title="dbt Cloud - Set Up a Repository" />
-</p>
+#### This code snippet is done using tabs in a resusable code snippet
 
-It should take a few seconds to create the repo and once it’s complete you’ll see a Successfully imported repository message and you’ll be able to click Continue. 
+<Snippet src="tutorial-managed-repo-and-initiate-project" />
 
-11. You can skip the final step of inviting other users to your account and click Skip & Complete to finish the setup. You’ll be able to go back and add users at any point within your account. 
+#### This code snippet is done using the WHCode component
 
-12. Congrats! Your account is officially set up and ready to rock. 
+<WHCode>
 
-## Start developing in dbt Cloud
+<p><code>I found some code</code></p>
+<p><code class="language-sql">select * from customers</code></p>
+<p>```I was hoping this code would be nicely formatted```</p>
+<p>Some text!</p>
```

Comment:
Hey @coapacetic, it looks like markdown code blocks require a line break above and below the snippet. Updating that line to have a space before and after should allow it to be formatted properly.

```suggestion
<p><code>I found some code</code></p>
<p><code class="language-sql">select * from customers</code></p>

<p>```I was hoping this code would be nicely formatted```</p>

<p>Some text!</p>
```

Reply:
![image](https://user-images.githubusercontent.com/13647110/164707736-23181a8c-056d-40c6-bab0-1f34bdd68d9a.png)

Reply:
thanks Jason!!!  I was able to hammer away at this a little bit more with Leona this afternoon and get things working.  Moving forward, the new component is `WHCode` for tabbed warehouse specific code snippets.  When we add additional warehouses, we can extend this to 5, 6, 7... tabs